File: Baekjoon/Basic/음계(2920).py
import sys
input = sys.stdin.readline

seq = list(map(int, input().split()))

def scanAsc(seq):
    prev = seq[0]
    for i in range(1, len(seq)):
        if prev > seq[i]:
            return 'mixed'
        prev = seq[i]
    return 'ascending'

def scanDsc(seq):
    prev = seq[0]
    for i in range(1, len(seq)):
        if prev < seq[i]:
            return 'mixed'
        prev = seq[i]
    return 'descending'

if (seq[0] < seq[1]):
    print(scanAsc(seq))
else:
    print(scanDsc(seq))

File: Baekjoon/Basic/음계(2920).js
/**
 * 
 * asc / dsc 판별
 * 
 */


const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on('line', (line) => {
    const seq = line.split(' ').map(Number);

    let result = '';
    if (seq[0] < seq[1]) {
        let prev = seq[1];
        for (let i=2; i<seq.length; i++) {
            if (seq[i] < prev) {
                console.log('mixed');
                process.exit();
            }
            prev = seq[i]
        }
        console.log('ascending');
    } else {
        let prev = seq[1];
        for (let i=2; i<seq.length; i++) {
            if (seq[i] > prev) {
                console.log('mixed');
                process.exit();
            }
            prev = seq[i];
        }
        console.log('descending');
    }

    process.exit();
});